packages/ripple-nuxt-tide/modules/news/tide.config.js

```diff
@@ -6,15 +6,10 @@ module.exports = {
     ],
 
     landingPage: [
-      'field_landing_page_component.field_paragraph_news_reference',
-      'field_landing_page_component.field_paragraph_news_reference.field_featured_image',
       'field_landing_page_component.field_paragraph_news_reference.field_featured_image.field_media_image',
-      'field_landing_page_component.field_paragraph_news_reference.field_topic',
-      'field_landing_page_component.field_paragraph_topic'
```

Comment:
'field_landing_page_component.field_paragraph_topic' is already in landing page module, no need to add again.